src/js/webSerial.js

```diff
@@ -30,12 +30,32 @@ class WebSerial extends EventTarget {
 
         this.logHead = "SERIAL: ";
 
+        this.port_counter = 0;
+        this.ports = null;
         this.port = null;
         this.reader = null;
         this.writer = null;
         this.reading = false;
 
         this.connect = this.connect.bind(this);
+
+        navigator.serial.addEventListener("connect", e => this.handleNewDevice(e.target).bind(this));
+        navigator.serial.addEventListener("disconnect", e => this.handleRemovedDevice(e.target).bind(this));
```

Comment:
Doesn't this work without `bind`?

Reply:
TBH, I'm not even sure what this is doing or how this even works.

Usually you bind to the function itself E.G.
```
this.handleNewDevice.bind(this)
```
But this seems to be binding to the return value of `handleRemoveDevice` and `handleNewDevice` I'm surprised it doesn't just crash

Reply:
<img width="603" alt="Screenshot 2024-05-15 at 18 35 51" src="https://github.com/betaflight/betaflight-configurator/assets/7774918/15845852-9243-425c-ac8e-9055580687a2">
yeah, this has an error

Reply:
I think that not, at least in my earlier tests, when the code was distinct. I will try again.

Reply:
Yo were right, it works without it. I remove it then.

Reply:
Fixed.